src/ListingPage.php

```diff
@@ -14,10 +14,16 @@ class ListingPage
      *
      */
     public array $context = [];
+    /**
+     * The maximum number of sticky posts to display.
+     *
+     * @const int MAX_STICKY_POSTS used to limit the number of sticky posts also to filter main query.
+     */
+    /** @static int */
+    public static int $MAX_STICKY_POSTS = 4;
```

Comment:
This isn't the "max" amount of sticky posts, right? It's the only amount that we show